msteams-platform/Teams-AI-library-tutorial.yml

```diff
@@ -165,49 +136,32 @@ items:
   durationInMinutes: 1
   content: |
 
-     Let's customize the prompt for your custom engine agent.
+   After you create your custom engine agent, let's configure it to interact with your data on OpenAI.
```

Comment:
interact with your data?

We are not using AI search here.

Reply:
Updated as suggested